All_Relationships/utils/disease_gene_lf.py

```diff
@@ -89,7 +89,7 @@ def LF_HETNET_DisGeNET(c):
 def LF_HETNET_GWAS(c):
```

Comment:
I would try to be more consistent about having comments for what each of the label functions are doing and also what the category is too